Dockerfile

```diff
@@ -38,21 +38,25 @@ RUN cmake --build . --config Release -j8
 
 FROM alpine
 
-RUN addgroup -S ucentralgw && adduser -S -G ucentralgw ucentralgw
+ENV UCENTRALGW_USER=ucentralgw \
+    UCENTRALGW_ROOT=/ucentralgw-data \
+    UCENTRALGW_CONFIG=/ucentralgw-data
+
+RUN addgroup -S "$UCENTRALGW_USER" && \
+    adduser -S -G "$UCENTRALGW_USER" "$UCENTRALGW_USER"
 
 RUN mkdir /ucentral
-RUN mkdir /ucentralgw-data
-RUN apk add --update --no-cache librdkafka mariadb-connector-c libpq unixodbc
+RUN mkdir -p "$UCENTRALGW_ROOT" "$UCENTRALGW_CONFIG"
+RUN apk add --update --no-cache librdkafka mariadb-connector-c libpq unixodbc su-exec
 
 COPY --from=builder /ucentralgw/cmake-build/ucentralgw /ucentral/ucentralgw
 COPY --from=builder /cppkafka/cmake-build/src/lib/* /lib/
 COPY --from=builder /poco/cmake-build/lib/* /lib/
 
-EXPOSE 15002
-EXPOSE 16002
-EXPOSE 16003
-EXPOSE 17002
-EXPOSE 16102
+EXPOSE 15002 16002 16003 17002 16102
+
+COPY docker-entrypoint.sh /
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
You can get rid of this line if you make `docker-entrypoint.sh` executable in the repository itself as permissions are also copied with file

Reply:
You're right, thought I set it to executable and Git didn't keep the permissions, but this was on another machine.